.github/workflows/build.yml

```diff
@@ -12,10 +12,17 @@ jobs:
       - name: checkout repo
         uses: actions/checkout@v2
 
+      - name: Read NodeJS version
+        id: nodejs-version
+        run: |
+          $content = Get-Content .\.nvmrc -Raw
```

Comment:
Much as it pains me to say it, we don't need a Windows CI runner here. We can build on ubuntu and speed up things here, while also keeping the shell languages to a minimum

Reply:
That is a good point - I don't remember why we picked Windows originally. That said I reckon we'd go back to Windows if/when we repurpose this action for publishing preview builds? https://github.com/hashicorp/vscode-terraform/issues/774

I _guess_ `vsce` works on other platforms as well, but I'm also _guessing_ that it works best on Windows? But these are all guesses honestly - I'm happy to do this on Linux - it would also probably save us some 💰  on GHA minutes.

Reply:
I can confirm from prior experience `vsce` works on nix as well as it does on windows. We wouldn't have to use Windows to publish preview or normal builds, the OS we build on doesn't affect the OS we deploy to.